### PR TITLE
fix: add npm install show error

### PR DESCRIPTION
**Pull request by xiaolang124**
1. When NPM install encounters error, show error notify message and add a report link.

image

image

### CODE REVIEW

File: packages/vscode-extension/src/debug/teamsfxTaskHandler.ts [hunk 1]
```diff
@@ -110,9 +112,25 @@ async function onDidEndTaskProcessHandler(event: vscode.TaskProcessEndEvent): Pr
...
+            task.name,
+            JSON.stringify(npmInstallLogInfo),
+            issueLink,
+            npmInstallLogInfo
+          ));
         }
-      } catch {
+      } catch (e) {
         // ignore any error
+        showError(new SystemError(
```
Comment (swatDong) [inline packages/vscode-extension/src/debug/teamsfxTaskHandler.ts:126]: This is internal error when getting log file, no need to show the same as above.
Telemetry could be a good improvement, but can be in later PR.

Comment (xiaolang124) [reply packages/vscode-extension/src/debug/teamsfxTaskHandler.ts]: Already remove inner error show error logic.

**Review by swatDong — APPROVED** [review]

**Comment by kimizhu** [comment]
This notification is too simple: Npm install encounters error.
It is better to add more info (at least to tell users the action item is to submit GitHub issue).

**Comment by kimizhu** [comment]
@xiaolang124 here is one example:
https://github.com/OfficeDev/TeamsFx/blob/bf681633893f2e170579d6e524354969310e824e/packages/fx-core/src/plugins/resource/bot/resources/messages.ts#L34

**Pull request merged**